testing/integration/tests/fil-malformed-syscall-actor/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
nit: toolchain versions should be locked at the same version as the rest of the project `nightly-2022-05-09`